aws-py-dynamicresource/requirements.txt

```
@@ -0,0 +1,4 @@
+pulumi>=2.0.0,<3.0.0
+pulumi-aws>=2.0.0,<3.0.0
```

We [recently updated](https://github.com/pulumi/examples/pull/761) all the `requirements.txt` in this repo to use version 3.x of the AWS provider.

```suggestion
pulumi-aws>=3.0.0,<4.0.0
```